CONTRIBUTING.md

```diff
@@ -2,18 +2,22 @@
 
 We would love for you to contribute to Reakit and help make it even better. By contributing to Reakit, you agree to abide by the [code of conduct](https://github.com/reakit/reakit/blob/master/CODE_OF_CONDUCT.md)
 
-- [Ownership](#ownership)
-  - [Why do we give out push access?](#why-do-we-give-out-push-access)
-  - [How can we help you get comfortable contributing?](#how-can-we-help-you-get-comfortable-contributing)
-  - [Our expectations on you as a contributor](#our-expectations-on-you-as-a-contributor)
-  - [What about if you have problems that cannot be discussed in a public issue?](#what-about-if-you-have-problems-that-cannot-be-discussed-in-a-public-issue)
-- [How to start contributing?](how-to-start-contributing)
-- [Contributing with code](#contributing-with-code)
-  - [Scripts](#scripts)
-  - [Creating components](#creating-components)
-  - [Working on website](#working-on-website)
-  - [Working on docs](#working-on-docs)
-- [Credits](#credits)
+- [Contributing to Reakit](#contributing-to-reakit)
```

Comment:
i just pressed save '-'

Reply:
I think that should be removed.

Do you have some plugin that generates table of contents?

Reply:
No i don't. I guess i'll manually edit that on vim...